### PR TITLE
Add deprecation warning and refer to NeurIPS demo

### PR DESCRIPTION
**Pull request by borzunov**
This PR adds a "**Note:**" below:

Screenshot 2021-12-21 at 18 30 31

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -261,6 +261,13 @@ <h1 class="faded title" style="margin-top:-3%;">
                     <p class="faded title" style="font-size:28px;">
                         What is hivemind?</p>
                     <br>
+                    <p class="faded text" style="margin-top: 16px; font-size: 18px;">
+                        <b style="color: #728FCE;">Note:</b>
+                         This page contains a severely outdated description of how hivemind works.
```
Comment (mryab) [inline index.html]:
```suggestion
                         This page contains a severely outdated description of what hivemind does.
```

File: index.html [hunk 2]
```diff
@@ -261,6 +261,13 @@ <h1 class="faded title" style="margin-top:-3%;">
...
+                         For the up-to-date information, see our NeurIPS 2021 demonstration
```
Comment (mryab) [inline index.html]:
```suggestion
                         For up-to-date information, see our NeurIPS 2021 demonstration
```

Comment (mryab) [inline index.html]: Alternatively,
```suggestion
                         For the most up-to-date information, see our NeurIPS 2021 demonstration
```

Comment (borzunov) [reply index.html]: Thanks, I've chosen the 1st option.

**Review by mryab — APPROVED** [review]

**Review by yhn112 — APPROVED** [review]
I'd like to say that the note is "too unnoticeable".
But this website seems rarely visited, so doesn't matter a lot.

**Pull request merged**